README.md

```diff
@@ -66,9 +83,9 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
-[] `$(".post").innerHTML`
-[] `$(".post").html()`
+[X] `$(".post").css("background", "peachpuff")`
+[X] `$(".post").innerHTML`
+[X] `$(".post").html()`
 [] `document.querySelectorAll(".post")[0].innerHTML`
```

Comment:
This is correct.  It calls "innetHTML" on the first ".post" element that was retrieved.